src/main/resources/assets/aquamirae/lang/en_us.json

```diff
@@ -199,17 +199,17 @@
   "effect.aquamirae.abyss_blindness.description": "Blinding Abyss armor effect. Decreases movement speed.",
   "effect.aquamirae.abyss_blindness.desc": "Blinding Abyss armor effect. Decreases movement speed.",
   "effect.aquamirae.armor_decrease": "§cPerforation",
-  "effect.aquamirae.armor_decrease.description": "Whisper of the Abyss effect. Reduces armor value.",
-  "effect.aquamirae.armor_decrease.desc": "Whisper of the Abyss effect. Reduces armor value.",
+  "effect.aquamirae.armor_decrease.description": "Whisper of the Abyss effect. Reduces armor value by 10% per level, up to 5 levels.",
+  "effect.aquamirae.armor_decrease.desc": "Whisper of the Abyss effect. Reduces armor value by 10% per level, up to 5 levels.",
   "effect.aquamirae.crystallization": "§cCrystallization",
-  "effect.aquamirae.crystallization.description": "Blinding Abyss armor effect. You deal 80% less damage and die when the effect ends.",
-  "effect.aquamirae.crystallization.desc": "Blinding Abyss armor effect. You deal 80% less damage and die when the effect ends.",
+  "effect.aquamirae.crystallization.description": "Blinding Abyss armor effect. Lowers attack damage by 80% and speed by 40%.",
```

Comment:
Why this change?

https://github.com/ObscuriaLithium/Aquamirae/blob/1.20.1-Forge-v6/src/main/java/com/obscuria/aquamirae/common/effects/CrystallizationMobEffect.java

there is no speed penalty and dying is very much a thing

Reply:
I'd like to mention that this PR is for the 1.20.1-Fabric-v6 branch. Testing the effect in game, it only gave me a damage and speed debuff; no death. In fact, you can see it for yourself here: https://github.com/ObscuriaLithium/Aquamirae/blob/1.20.1-Fabric-v6/src/main/java/com/obscuria/aquamirae/common/effects/CrystallizationEffect.java

Quite interesting that the mob effect has different debuffs between mod loaders.